inc/ticket.class.php

```diff
@@ -5100,12 +5100,13 @@ static function showCentralList($start, $status = "process", $showgrouptickets =
             );
             break;
 
-         case "process" : // planned or assigned tickets
+         case "process" : // planned or assigned or incoming tickets
             $WHERE = array_merge(
                $WHERE,
                $search_assign,
-               ['glpi_tickets.status' => self::getProcessStatusArray()]
```

Comment:
I'd prefer to keep the call to the `getProcessStatusArray`, in case it change in the future.

Reply:
Fixed